account_payment_order_return/wizard/account_payment_line_create.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class AccountPaymentLineCreate(models.TransientModel):
+    _inherit = 'account.payment.line.create'
+
+    avoid_returned = fields.Boolean(
```

Comment:
Make the contrary logic: put a field called `include_returned` and unless this is checked, you include in the domain the new condition.

Reply:
Done